infrastructure/local.tf

```diff
@@ -0,0 +1,23 @@
+locals {
+  org              = "pins"
+  service_name     = "appeals-bo"
+  primary_location = "uksouth"
```

Comment:
Should this be `uk-south`?
https://github.com/claranet/terraform-azurerm-regions/blob/master/REGIONS.md

Reply:
Yes good spot - I'm sure I ran a plan and it didn't complain!